packages/eslint-plugin-pf-codemods/src/rules/v6/accordionItemWarnUpdateMarkup/accordionItem-warn-update-markup.ts

```diff
@@ -17,10 +17,7 @@ module.exports = {
       ? {}
       : {
           JSXOpeningElement(node: JSXOpeningElement) {
-            if (
-              node.name.type === "JSXIdentifier" &&
-              accordionItemImport.local.name === node.name.name
-            ) {
+            if (checkMatchingJSXOpeningElement(node, accordionItemImport)) {
```

Comment:
Only updated the one file for now. I could update the rest of the files as part of this or we can do that as a followup at some other point, I don't mind either way

Reply:
I'm also good either way you want to go about it.

Reply:
I don't think we need to refactor our existing codemods, since they are a "one time use" code and we probably won't copy the code from them in the future. So IMO updating the `write-rule` generator is sufficient.